controllers/mailListController.js

```diff
@@ -49,6 +60,10 @@ MailListController.add = function(req, res){
     let email = req.body.email;
     let mailchimpGroup = req.body.group;
 
+    // TODO: Is there some better place/way to do this? The MDN indicates we need 
+    // to set this header on both the preflight request and the actual, _real_ request. 
+    res.setHeader('Access-Control-Allow-Origin', ALLOWED_ORIGIN); 
```

Comment:
You might be able to interact with the res's header in server.js itself. I put a suggestion in server.js.